test/e2e_node/runner/remote/run_remote.go

```diff
@@ -119,11 +129,12 @@ type internalImageConfig struct {
 }
 
 type internalGCEImage struct {
```

Comment:
Doesn't need to happen immediately, but often its easier to move things around when you still have a small surface area.